rkt/api_service_stub.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !sdjournal
+
+package main
+
+import (
+	"fmt"
+
+	"github.com/coreos/rkt/api/v1alpha"
+)
+
+func (s *v1AlphaAPIServer) constrainedGetLogs(request *v1alpha.GetLogsRequest, server v1alpha.PublicAPI_GetLogsServer) error {
+	return fmt.Errorf("unsupported logging subsystem")
```

Comment:
This error is more useful for console output rather than an err equality check like `io.EOF`. I'd keep them as separate errors.